backend/src/test/java/com/github/martingaston/application/http/ResponseTest.java

```diff
@@ -22,6 +22,16 @@ void builds200Response() {
             assertThat(response.headers().get("Content-Length")).isEqualTo("11");
         }
 
+        @Test
+        void buildsJsonResponse() {
```

Comment:
The tests description could be more specific here, you are testing that `.json` set the response header content type as `application/json`